src/Microsoft.Restier.Core/Conventions/ConventionBasedEntitySetFilter.cs

```diff
@@ -22,23 +22,33 @@ private ConventionBasedEntitySetFilter(Type targetType)
             this.targetType = targetType;
         }
 
+        public IQueryExpressionFilter Inner { get; set; }
+
         /// <inheritdoc/>
         public static void ApplyTo(
             IServiceCollection services,
             Type targetType)
         {
             Ensure.NotNull(services, "services");
             Ensure.NotNull(targetType, "targetType");
-            services.CutoffPrevious<IQueryExpressionFilter>(new ConventionBasedEntitySetFilter(targetType));
+            services.ChainPrevious<IQueryExpressionFilter>(next => new ConventionBasedEntitySetFilter(targetType)
```

Comment:
I see, I will make it as next TODO item and make changes to other convention based filter.